### ISSUE TITLE
Allow multiple losses per output

### ISSUE THREAD

**Issue by cooperlab** [PathologyDataScience/glimr#2]
Currently `glimr.keras.keras_losses` only allows one loss per output. Enable multiple losses per output, similar to `glimr.keras.keras_metrics`.
